README.md

```diff
@@ -32,6 +56,46 @@ github "rudderlabs/rudder-sdk-ios" "v1.1.4"
 #import <Rudder/Rudder.h>
 ```
 
+### Swift Package Manager (SPM)
+
+You can also add the RudderStack iOS SDK via Swift Package Mangaer, via one of the following two ways:
+
+* [Xcode](#xcode)
+* [Swift](#swift)
+
+#### Xcode
+
+* Go to **File** - **Swift Packages** - **Add Package Dependency...**, as shown:
+
+* Enter the package repository (git@github.com:rudderlabs/rudder-sdk-ios.git) in the search bar, as shown:
+
+* In **Choose Package Options**, go to **Rules**. Select the **Version** as **Up to Next Major** and enter as the value, as shown:
+
+#### Swift
+
+To leverage package.swift, use the following snippet in your project:
+
+```swift
+import PackageDescription
+
+let package = Package(
+    name: "RudderApp",
+    dependencies: [
+        // Add a package containing Analytics as the name along with the git url
+        .package(
+            name: " ",
+            url: ""
```

Comment:
@pallabmaiti Should we prefill the URL here?

Reply:
Yes. I think they are still woking on it.

Reply:
@arnab-p this is incomplete. I'm waiting for some inputs from Abhishek before we proceed. Need to add screenshots as well.

Reply:
I'll ping both of you once the PR is ready to review. Changed the status to WIP for now.